tests/jobs.py

```diff
@@ -19,12 +19,14 @@
 
```

Comment:
Do you think we should add tests for the new functions? Seems like it could be dangerous to modify them otherwise.